index.js

```diff
@@ -67,6 +67,7 @@ module.exports = {
     self.addToAdminBar();
     self.apos.pages.addAfterContextMenu(self.menu);
     self.enableHelpers();
+    self.enablecrossDomainSessionCache();
```

Comment:
I think you noted this already, but better camelCasing here: `enableCrossDomainSessionCache();`